src/Instrumentation/ExtRdKafka/composer.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "open-telemetry/opentelemetry-auto-ext-rdkafka",
+  "description": "OpenTelemetry auto-instrumentation for psr18 http clients.",
+  "keywords": ["opentelemetry", "otel", "open-telemetry", "tracing", "ext-rdkafka", "instrumentation"],
+  "type": "library",
+  "homepage": "https://opentelemetry.io/docs/php",
+  "readme": "./README.md",
+  "license": "Apache-2.0",
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "require": {
+    "php": "^8.0",
```

Comment:
the tests only run on 8.2+ (because of exclusions added the matrix in .github/workflows/php.yml) - so this should either be 8.2 or the tests should run on 8.0+